examples/meta/src/features/string_char.sg

```diff
@@ -0,0 +1,11 @@
+File words = csv_file("../../data/words.dat")
+
+#![create_features]
+Features f = string_features(words, enum EAlphabet.RAWBYTE)
+#![create_features]
+
+#![output stat]
+int max_string_length = f.get_int("max_string_length")
+int number_of_strings = f.get_int("get_num_vectors")
+CharVector features = f.get_char_string_list("get_features")
```

Comment:
I think `java.json` needs to be updated to that it doesn't translate `DoubleMatrix` ... java swig in fact has typemaps for `SGStringList`, i.e. it is able to translate this straight to a java data structure, done [here](https://github.com/shogun-toolbox/shogun/blob/develop/src/interfaces/java/swig_typemaps.i#L674):

Noticed that when staring the the CI error [here](https://dev.azure.com/shogunml/shogun/_build/results?buildId=290)